tf2onnx/graph.py

```diff
@@ -1258,10 +1258,27 @@ def insert_new_node_on_input(self, node, op_type, input_name, name=None, domain=
                 break
         return new_node
 
-    def insert_new_node_on_output(self, op_type, output_name, name=None, inputs=None, domain=None, **kwargs):
+    def insert_node_on_output(self, node, output_name=None):
+        """
+        The inserted node takes the *output_name* as input and produces a
+        new output. The function goes through every node taking *output_name*
+        and replaces it by the new output name.
+        """
+        if output_name is None:
+            output_name = node.input[0]
+        new_output = node.output[0]
+
+        to_replace = [self.get_node_by_name(n) for n in self._output_to_consumers[output_name]]
+        to_replace = [n for n in to_replace if n != node]
+        self.replace_all_inputs(output_name, new_output, ops=to_replace)
+        return node
+
+    def insert_new_node_on_output(self, op_type, output_name=None, name=None, inputs=None, domain=None, **kwargs):
         """Create and insert a new node into the graph.
+        It then calls insert_node_on_output.
+
         Args:
-            op_type: type for new operation
+            op_type: type for new operation or an existing node
```

Comment:
```suggestion
            op_type: type for new operation
```

Reply:
I think you forgot to change this back.

Reply:
I forgot to remove the comment, otherwise, I just split the function in two.